src/main/webapp/app/modules/provider-site/provider-site.tsx

```diff
@@ -0,0 +1,155 @@
+import 'react-toastify/dist/ReactToastify.css';
+import './provider-site.scss';
+import '../../bootstrap.scss';
+
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { IRootState } from 'app/shared/reducers';
+import { getSession } from 'app/shared/reducers/authentication';
+import { getProfile } from 'app/shared/reducers/application-profile';
+import { setLocale } from 'app/shared/reducers/locale';
+import { hasAnyAuthority } from 'app/shared/auth/private-route';
+import { AUTHORITIES } from 'app/config/constants';
+import { toast, ToastContainer, ToastPosition } from 'react-toastify';
+import ErrorBoundary from 'app/shared/error/error-boundary';
+import Header from './menus/header';
+import HeaderMobile from 'app/modules/provider-site/mobile-components/header-mobile';
+import { Avatar } from './avatar';
+import UserRecords from './user-records';
+import { translate, Translate } from 'react-jhipster';
+import { Home } from 'app/modules/home/home';
+import { RouteComponentProps } from 'react-router-dom';
+import { getUser } from 'app/modules/administration/user-management/user-management.reducer';
+import { SideMenu } from './mobile-components/side-menu';
+import MediaQuery from 'react-responsive';
+
+export interface IProviderSiteProps extends StateProps, DispatchProps, RouteComponentProps<{ url: string }> {}
+
+export interface IProviderSiteState {
+  menuOpen: boolean;
+}
+
+export class ProviderSite extends React.Component<IProviderSiteProps, IProviderSiteState> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      menuOpen: false
+    };
+  }
+
+  componentDidMount() {
+    this.props.getUser(this.props.account.login);
+    this.props.getSession();
+    this.props.getProfile();
+  }
+
+  toggleMenu = () => {
+    this.setState({ menuOpen: !this.state.menuOpen });
+  };
+
+  render() {
+    const { userLogin } = this.props;
+    const { menuOpen } = this.state;
+
+    const HeaderComponent = (
+      <div>
+        <MediaQuery maxDeviceWidth={768}>
```

Comment:
What will happen if the width will be 768? maxDeviceWidth and minDeviceWidth are the same. Also what happens in a horizontal view?

Reply:
You are right, I'll change this